### ISSUE TITLE
Map Data overlay blocks browser if zoomed out to a large map area

### ISSUE THREAD

**Issue by AxelBoldt** [openstreetmap/openstreetmap-website#3243]
If the Map Data overlay is inadvertently used while being zoomed out to a large map area, the browser crawls to a halt. This can happen, for instance, if the user forgets to switch off the Map Data overlay after use: the website remembers the setting for the next session, which might start off with a large area. 

I'm not sure how to properly solve the issue; maybe simply disable the Map Data overlay whenever the current zoom level is too high?

**Comment by tomhughes** [comment]
It already is disabled at low zoom, though the cutoff seems to be 12 which is lower than I was expecting.

**Comment by AxelBoldt** [comment]
You are right. Maybe the the OSM website Map Data overlay should use the same zoom cutoff as the iD editor?

I just noticed that the website initially won't show any Map Data but displays a warning "Loading 4069 features, which may make your browser slow or unresponsive. Are sure you want to display this data?" Even so, the browser is stuck and I can't click on anything.

**Comment by 1ec5** [comment]
In the future, replacing the map data overlay with Spyglass would eliminate the need to disable it at low zoom levels. We don’t have an issue tracking that replacement yet, but openstreetmap/operations#1313 tracks setting up our own instance of Spyglass first.
